action.yml

```diff
@@ -14,6 +14,14 @@ inputs:
   anaconda_nightly_upload_token:
     description: 'Token to upload to scientific python org'
     required: true
+  anaconda_nightly_upload_organization:
+    description: 'Organisation name to upload the wheels to'
+    required: false
+    default: scientific-python-nightly-wheels
+  anaconda_nightly_upload_labels:
+    description: 'Labels assigned to the uploaded artifacts'
+    required: false
+    default: [main]
```

Comment:
@skoudoro Hey sorry to have one final question here (now that I've arrived), but can you point me to a reference on how you know that `anaconda upload --label` will accept a list of labels (as opposed to repeated calls of `--label` like how you would have to do with `--channel` with `micromamba install`? `micromamba install --channel main --channel conda-forge numpy`)?

The `anaconda-client` docs are not good, so it is possible I'm missing something. Have you tested this with any uploads to your registry yet?

Reply:
> can you point me to a reference on how you know that anaconda upload --label will accept a list of labels

As you can see below, I just used the help of the command line (`anaconda upload --help`.

![image](https://github.com/scientific-python/upload-nightly-action/assets/23106443/1d63860f-9504-485b-8af9-5770f43554a1)

There is also the following documentation : https://docs.anaconda.com/anaconda-repository/user-guide/tutorials/#using-labels-in-the-development-cycle

>  Have you tested this with any uploads to your registry yet?

Yes,  I tried with some artifacts on our repository. I use the following command manually:
`anaconda upload --force --user dipy --label dev --label main  *1.6.0*.whl`

Reply:
Not sure how to handle the list with the docker. I am looking forward for your feedback

Reply:
yes, I just updated the code. This is not a list, sorry for the confusion.

>  how you're using labels now and why you want to have multiple labels?

- We use only the label dev for now
- in our case (https://anaconda.org/dipy/dipy), the organization name is not explicit for nightly wheels. I would like to avoid random people downloading nightly wheels and believing that is it a release. So, we tag them as dev instead of main. it force user to be explicit and conscious of what they are trying to get. At the same time, it is not an issue for downstream packages.
- we do not use multiple labels as the same time. the line above was just an example to show you how it can be done.